README.md

```diff
@@ -1 +1,62 @@
 # Manifest Schema
+
+This project contains publicly available json schemas that define slacks `manifest.json` file. JSON schemas can be used for a number of tasks but the purpose of this specific schema is to provide developers validation and prediction capabilities while editing `manifest.json` files in IDEs.
+
+**Guide Outline**:
+
+- [Manifest Schema](#manifest-schema)
+  - [Usage](#usage)
+  - [Project Structure](#project-structure)
+    - [`manifest.schema.json`](#manifestschemajson)
+    - [`/schemas`](#schemas)
+    - [`/tests`](#tests)
+    - [`/scripts`](#scripts)
+  - [Resources](#resources)
+
+---
+
+## Usage
+
+JSON schemas can be used for various tasks, the goal of this project is to provide a JSON schema that defines slacks `manifest.json` and provides validation and IntelliSense capabilities to developers editing `manifest.json` files in IDEs that support these features.
+
+<img src="https://user-images.githubusercontent.com/25348381/195903601-ab1e200a-53b0-470a-9374-bad8b93b35fd.gif" width="60%"/>
+
+To get these capabilities simply include the schema in a JSON file
+
+```json
+{
+  "$schema": "https://raw.githubusercontent.com/slackapi/manifest-schema/main/manifest.schema.json",
+  "_metadata":{
+    "major_version": 2
+  },
+  "settings": { ... }
+  ...
+}
+```
+
+## Project Structure
+
+### [`manifest.schema.json`](manifest.schema.json)
```

Comment:
I don't think this header link links to anything.

Reply:
This links to the file in the project 😎

Reply:
Ah yes derp